### PR TITLE
NAS-115200 / 22.12 / NAS-115200: Refactoring OpenvpnServerComponent to ix-forms

### PR DESCRIPTION
**Pull request by undsoft**
Testing: Network -> OpenVPN -> Server.
You may also need to create certificates in Credentials -> Certificates. You can use appropriate profile when creating a certificate.

### CODE REVIEW

File: src/app/pages/network/components/download-client-config-modal/download-client-config-modal.component.spec.ts [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+      },
+    ],
+  });
+
+  beforeEach(() => {
+    spectator = createComponent();
+    loader = TestbedHarnessEnvironment.loader(spectator.fixture);
+  });
+
+  it('shows a list of client certificates', async () => {
```
Comment (RehanY147) [inline src/app/pages/network/components/download-client-config-modal/download-client-config-modal.component.spec.ts:54]: This test should be fixed too.

**Review by RehanY147 — CHANGES_REQUESTED** [review]
The `ix-select` on clicking `Download Client Config` shows no options and is empty. Previously, it showed a list of certificates available.

File: src/app/pages/network/components/download-client-config-modal/download-client-config-modal.component.spec.ts (hunk 1 above)
Comment (undsoft) [reply src/app/pages/network/components/download-client-config-modal/download-client-config-modal.component.spec.ts:51]: Thanks, fixed.

**Review by RehanY147 — APPROVED** [review]
Looks good to me

**Pull request merged**